llvm/utils/lit/lit/llvm/config.py

```diff
@@ -108,14 +118,17 @@ def __init__(self, lit_config, config):
         elif platform.system() == "OS/390":
             features.add("system-zos")
 
-        # Native compilation: host arch == default triple arch
+        # Native compilation: host arch == default triple arch (sans vendor)
```

Comment:
Without vendor assuming the vendor is not significant to ABI.

Maybe you could say:
Native compilation: host arch == default triple arch (for ABI purposes)